testdata/go/v4-alpha/memcached-operator/test/e2e/e2e_test.go

```diff
@@ -192,5 +227,116 @@ var _ = Describe("memcached", func() {
 			}
 			Eventually(getStatus, time.Minute, time.Second).Should(Succeed())
 		})
+
+		Context("Memcached Operator metrics", func() {
+			var portForwardCmd *exec.Cmd
+			var sourcePort int
+
+			BeforeEach(func() {
+				var err error
+				rand.Seed(time.Now().UnixNano())
+				sourcePort = 4321 + rand.Intn(6000)
+				targetPort := 8080
+
+				By(fmt.Sprintf("issuing a port-forwarding command to access Memcached operator metrics endpoint on port %d", sourcePort))
+				portForwardCmd, err = startPortForwardCmd(sourcePort, targetPort)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			AfterEach(func() {
+				By("removing the port-forwarding command")
+				Expect(killPortForwardCmd(portForwardCmd)).To(Succeed())
+			})
```

Comment:
Why not do it using the metrics endpoint and curling the result. see: 

- https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v3/plugin_cluster_test.go#L264-L272
- https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v3/plugin_cluster_test.go#L281-L294
- https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v3/plugin_cluster_test.go#L359-L364

Reply:
Not sure if that's exactly what you meant, but we do curl the metrics endpoint here. Please see https://github.com/operator-framework/operator-sdk/blob/61dcef47b5f924ca240d0fed585927ccae245e7e/testdata/go/v4-alpha/memcached-operator/test/e2e/e2e_test.go#L271

Reply:
@assafad you are using portForward. Please, check how we do that in the other tests and I think we should to do at the same way.